signer/signer_trust.go

```diff
@@ -30,10 +31,14 @@ type NotarySigner struct {
 func NewNotarySigner(hostname string, port string, tlscafile string) *NotarySigner {
 	var opts []grpc.DialOption
 	netAddr := net.JoinHostPort(hostname, port)
-	creds, err := credentials.NewClientTLSFromFile(tlscafile, hostname)
+	tlsConfig, err := utils.ConfigureClientTLS(&utils.ClientTLSOpts{
+		RootCAFile: tlscafile,
```

Comment:
credentials.NewClientTLSFromFile supports multiple CAs in a single file; this (through trustmanager.LoadCertFromFile explicitly (and silently!) throws away any certificates after the first one.

Does this difference matter?